frontend/portals/SearchSuggestions/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, {
+  memo, useEffect, useMemo, useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
+import { useRoute } from '@shopgate/engage/core';
```

Comment:
there is a small gap where the underlying content shows through
![image](https://user-images.githubusercontent.com/5811286/82347888-78739000-99f8-11ea-9f3a-affc6d6d0418.png)